src/python/module/smartredis/error.py

```diff
@@ -61,6 +61,8 @@ def _check_error(cpp_error: str, method: str = "", key: str = "") -> str:
         msg = ""
         if method:
             msg = f"{method} execution failed\n"
+        if key:
+            msg += f"key: {key}\n"
```

Comment:
This will usually be redundant with the underlying Redis error message, so I think this change should not happen

Reply:
the linter complains about the unused variable. can i get rid of the unused method param, too?

Reply:
Let's check to see if anyone ever passes the key to the constructor first. If not (which I expect), then definitely

Reply:
This is the only change that I'm really concerned about btw

Reply:
Searched for any usage of the parameter. Found none in smartredis or smartsim. Removed parameter.